### PR TITLE
Refactor: Split Editor Wrapper into multiple components

### PR DESCRIPTION
**Pull request by max-nextcloud**
`EditorWrapper` is fairly large and hard to understand. It handles almost all editor related things - Sessions, Sync, Document, EditorLayout, Menubars etc.

In order to rework the collaboration it might be helpful to split it into smaller pieces so it becomes easier to understand and handle.

`Editor/Status` handles the status display in the menubar.

### CODE REVIEW

**Comment by max-nextcloud** [comment]
This is just some preparation - but i think it's helpful in its own right and changing quite a lot of file structures - so maybe good to get it in already.

**Comment by juliushaertl** [comment]
@max-nextcloud Setting labels is highly appreciated ;)

**Review by juliushaertl — APPROVED** [review]

File: src/components/Editor/Main.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	name: 'Main',
+	components: {
+		MediaHandler,
+	},
+
+}
+</script>
+
+<style scoped lang="scss">
+	.text-editor__main, .editor {
```
Comment (vinicius73) [inline src/components/Editor/Main.vue:43]: We do not have a `.editor` element in this component.

**Review by vinicius73 — APPROVED** [review]

**Pull request merged**